src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -2263,6 +2263,11 @@ namespace winrt::TerminalApp::implementation
                 // Force the TerminalTab to re-grab its currently active control's title.
                 terminalTab->UpdateTitle();
             }
+
+            if (auto settingsTab = tab.try_as<TerminalApp::SettingsTab>())
```

Comment:
`else if`?

Maybe in the future we should try and combine these two so it's just `ITab::UpdateSettings`, each with their own implementation, but this is probably fine for now.

Reply:
(this is a nit)

Reply:
This is a really good idea. Filed #8452 for tracking.

Reply:
Also, fixed the else if thing.